### ISSUE TITLE
Is this a way to change to locale of the verified pin or other parameters in the library?

### ISSUE THREAD

**Issue by proson1** [Nexmo/nexmo-php#119]
as subject.

**Comment by lornajane** [comment]
I'm sorry, I don't understand the question.  What is "this" in this context please? If you can tell us what you are trying to do, and perhaps share the code, we will do our best to help.

**Comment by proson1** [comment]
OK let me rephrase the question, I want to change the locale of the verified pin statement from English to Chinese but don't know how to do it. 

Thanks!

**Comment by lornajane** [comment]
That sounds like you are using the Verify API, check out the [API docs](https://developer.nexmo.com/api/verify), particularly for the `lg` parameter which takes any of our [supported language codes](https://developer.nexmo.com/api/verify). Does that answer your question?

**Comment by proson1** [comment]
Well I'm using the PHP API and I see a method call setLanguage, and I tried to call like this:

```
$V = $verification->setLanguage('zh-tw');

  	$id = $v->getRequestId();
```
But it returns me the English text.

**Comment by lornajane** [comment]
The parameter needs to be set when you start the verification process as shown in the API documentation (http://localhost:3000/api/verify).  You didn't show your code but you probably have a `$client->verify->start()` call and the `lg` parameter should be included here, something like this:

```
$verification = $client->verify()->start([
    'number' => '447700900000',
    'brand'  => 'My App',
    'lg' => 'zh-tw'
]);
```

The docs also have a note about `zh-tw` producing an SMS in Chinese but the phone call in English (see http://localhost:3000/api/verify#language-codes) so look out for that too.

**Comment by lornajane** [comment]
Closing due to inactivity but let us know if you need anything else!
